lib/nimble_template/addons/github.ex

```diff
@@ -128,4 +134,18 @@ defmodule NimbleTemplate.Addons.Github do
 
     project
   end
+
+  defp append_wiki_into_readme(project) do
+    Generator.append_content(
+      "README.md",
+      """
+
+      ## Project documentation
+
+      Most of the documentation locate in the `.github/wiki` directory, which is published to the [project's Github wiki](https://github.com/[REPO]/wiki).
```

Comment:
```suggestion
      Most of the documentation is located in the `.github/wiki` directory, which is published to the [project's Github wiki](https://github.com/[REPO]/wiki).
```

Reply:
Thanks Terone, I have updated it in https://github.com/nimblehq/elixir-templates/pull/149/commits/a585cf3ceb677759e2807d77521d09fa37982aa4